assets/js/admin/admin.js

```diff
@@ -296,6 +296,33 @@ jQuery( document ).ready( function( $ ) {
 			$firstError = $firstError ? $firstError : $row;
 		}
 
+		/**
+		 * Add invalid URL flag to a form field.
+		 *
+		 * @param {jQuery} $row
+		 * @param {jQuery} $field
+		 */
+		function addUrlError( $row, $field ) {
+			const $label = $row.find( '.cmb-th label' );
+
+			$errorFields.push(
+				{
+					id: $row.hasClass( 'cmb-repeat' ) ? `${$label.attr( 'for' )}_repeat` : $label.attr( 'for' ),
+					label: $label.text(),
+					type: 'url'
+				}
+			);
+			if ( $row.hasClass( 'cmb-repeat' ) ) {
+				$field.parent( '.cmb-td' ).parent( '.cmb-repeat-row' ).addClass( 'form-invalid' );
+			} else {
+				$row.addClass( 'form-invalid' );
```

Comment:
Class `form-invalid` gets applied to Permanent Link field if other errors exist and Permanent Link field is valid. See https://github.com/platform-coop-toolkit/learning-commons-framework/issues/155

Reply:
I tried to reproduce this and I couldn't. Did you run `npm run build` to make sure the latest version of the validation script is present after checking out this branch? I'm wondering if that might be the issue.

Reply:
My apologies. I was running the fix/url-validation branch without having merged changes from the dev branch. Merging dev and running npm run start seems to have resolved this issue I was seeing.